fields/types/date/DateFilter.js

```diff
@@ -102,7 +102,7 @@ var DateFilter = React.createClass({
     });
   },
   switchBetweenActiveInputFields(e, day, modifiers) {
-    if (modifiers.indexOf('disabled') > -1) return;
```

Comment:
Just curious. What is the benefit for the style changing?

Reply:
It causes a bug here since the `modifiers` is not an array.
Also, retain the ability to disable date picker by assigning the disable attribute in modifiers.